keps/sig-architecture/3935-oldest-node-newest-control-plane/README.md

```diff
@@ -0,0 +1,729 @@
+    Qualify 1.43/1.40                                 : n3_143qual3,       2031-04-01,4w
+    Qualify 1.43                                      : n3_143qual,        2031-04-01,4w
+    Control Plane → v1.41 → v1.42 → v1.43             : n3_143cpupgrade,   after n3_143qual, 1d
+    Nodes → v1.43 (drain/recreate workloads)          : n3_143nodeupgrade, after n3_143cpupgrade,  1w
+    Run v1.43...                                      : n3_143run,         after n3_143nodeupgrade, 2031-07-01
+
+    section with n-2
+    Qualify 1.40                                      : n2_140qual,        2030-04-01,4w
+    Run v1.40                                         : n2_140run,         after n2_140qual, 1y
+    Qualify 1.41/1.40                                 : n3_143qual1,       2031-04-01,4w
+    Qualify 1.42/1.40                                 : n3_143qual2,       2031-04-01,4w
+    Qualify 1.42                                      : n3_143qual3,       2031-04-01,4w
+    Qualify 1.43/1.42                                 : n3_143qual4,       2031-04-01,4w
+    Qualify 1.43                                      : n2_143qual,        2031-04-01,4w
+    Control Plane → v1.41 → v1.42                     : n2_142cpupgrade,   after n2_143qual, 1d
+    Nodes → v1.42 (drain/recreate workloads)          : n2_142nodeupgrade, after n2_142cpupgrade,  1w
+    Control Plane → v1.43                             : n2_143cpupgrade,   after n2_142nodeupgrade,  1d
+    Nodes → v1.43 (drain/recreate workloads)          : n2_143nodeupgrade, after n2_143cpupgrade,  1w
+    Run v1.43...                                      : n2_143run,         after n2_143nodeupgrade, 2031-07-01
+```
+-->
+
+### Goals
+
+* Allow users to take full advantage of the [yearly support period](https://github.com/kubernetes/enhancements/tree/master/keps/sig-release/1498-kubernetes-yearly-support-period)
+* Improve upgrade paths for nodes / node pools to minimize required workload churn
+* Validate oldest supported node components work with latest control plane components
+* Update the [Kubernetes version skew policy](https://kubernetes.io/releases/version-skew-policy/) to indicate node components from the oldest supported minor version work with control plane components from the newest supported minor version, starting with Kubernetes `v1.<TBD>` nodes.
+
+### Non-Goals
+
+* This does not propose any changes to the number of supported minor versions
+* This does not propose any changes to the minor version support duration
+* This does not propose any changes to the Kubernetes release process
+* This does not propose any changes to the mechanics control plane components use to enable features and work with older nodes without features ([KEP-3920](https://github.com/kubernetes/enhancements/pull/3920) could help with this)
+* This does not propose delaying introduction of existing features SIGs have planned around existing n-2 node/control-plane skew
```

Comment:
Are there any KEPs related to this that you are aware of, or is this still in the discussion phase (curious to learn more about this if anything exists). thank you!